Core/include/Acts/TrackFitting/GlobalChiSquareFitter.hpp

```diff
@@ -753,6 +724,11 @@ class Gx2Fitter {
         break;
       }
 
+      // TODO investigate further
+      if (chi2sum > oldChi2sum + 1e-5) {
```

Comment:
1e-5 is a rather small value, I think we could even live with a delta(abs(chi2)) change as convergence criteria.

Reply:
Sounds good. For convergence I now use `rel 1e-11` for my testing.
The `1e-5` above is for investigating if the convergence is more or less monotonic.